sentry.conf.py

```diff
@@ -304,3 +304,20 @@
 if 'BITBUCKET_CONSUMER_KEY' in os.environ:
     BITBUCKET_CONSUMER_KEY = env('BITBUCKET_CONSUMER_KEY')
     BITBUCKET_CONSUMER_SECRET = env('BITBUCKET_CONSUMER_SECRET')
+
+## Custom hint modifications
+
+# Use 3 web workers
+SENTRY_WEB_OPTIONS = {
+    'workers': 3,  # the number of web workers
+}
+
+# Force ssl when using ssl
```

Comment:
I definitely disagree with these changes in the config file. We don't not support `django-sslify` nor do we want this installed as a default which we, the Sentry team, don't use ourselves.